gprofiler/python.py

```diff
@@ -92,6 +92,12 @@ def _find_python_processes_to_profile(self) -> List[Process]:
                 if any(item in cmdline for item in self.BLACKLISTED_PYTHON_PROCS):
                     continue
 
+                # PyPy is called pypy3 or pypy (for 2)
+                # py-spy is, of course, only for CPython, and will report a possibly not-so-nice error
+                # when invoked on pypy.
+                if os.path.basename(process.exe()).startswith("pypy"):
```

Comment:
no way of other process exe with pypy compiled with (DSO or statically)? 
at least add a comment.

Reply:
There is -
```
root@915ff1b1a217:/# ldd /opt/pypy/bin/pypy3
	linux-vdso.so.1 (0x00007fffc6ff5000)
	libpypy3-c.so => /opt/pypy/bin/libpypy3-c.so (0x00007f7446e7d000)
```

I'm not aware of anything directly loading this `libpypy` tho. So until proven relevant, I'll add a comment. If we ever encounter it, we can change this code to check the maps.